config/settings.yml

```diff
@@ -4,6 +4,9 @@ application:
 
 analytics:
   web_property_id: ~
+  pkcs12_key: ~
```

Comment:
Can you document the expected type here?  I'm guessing this is a URI to a key that should be in a private s3 bucket?

Reply:
Added a comment above the block.